src/mozilla_sec_eia/ex_21/create_labeled_dataset.py

```diff
@@ -72,7 +73,7 @@ def create_inputs_for_label_studio(
             "id": f"{filename_no_ext}",
             "data": {"ocr": f"gs://labeled-ex21-filings/unlabeled/{image_filename}"},
             "annotations": [],
-            "predictions": [{"model_version": "v1.0", "result": []}],
+            "predictions": [{"model_version": f"{model_version}", "result": []}],
```

Comment:
This is kind of annoying, but it's important to specify a model version when making a new version of label studio inputs, else everything gets all messed up when you sync GCS with Label Studio.